frontend/src/pages/About/About.js

```diff
@@ -0,0 +1,21 @@
+import { Main } from '/src/components'
+
+const About = () => {
+  return <Main>
+    <Main.Header center />
+    <h2>About</h2>
+    <p>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Varius vestibulum pretium velit vitae euismod magna mi, in. Erat nulla risus, tellus diam. Faucibus sagittis parturient convallis egestas duis. Condimentum dictumst habitant penatibus egestas sem scelerisque. Ornare dui, est senectus metus morbi. Vulputate maecenas suspendisse ornare urna, feugiat elit ornare lectus ornare. Adipiscing nunc iaculis tortor felis amet lacus. Consectetur fermentum eget aliquet nulla pellentesque enim elementum mattis sed.<br/><br/>
```

Comment:
Hm I like this copy but it's a little hard to read, maybe it's the latin? It might just be me...